chia/cmds/wallet_funcs.py

```diff
@@ -232,7 +232,10 @@ async def execute_with_wallet(wallet_rpc_port: int, fingerprint: int, extra_para
         pass
     except Exception as e:
         if isinstance(e, aiohttp.ClientConnectorError):
-            print(f"Connection error. Check if wallet is running at {wallet_rpc_port}")
+            print(
+                f"Connection error. Check if wallet is running at {wallet_rpc_port}. "
+                f"You can run the wallet by:\n    chia start wallet"
```

Comment:
the prefix `f` in the second line here is redundant. The four blanks are optionally a `\t`. The "by" is probably rather a "via". I also note that the first line swallows the "the" in front of the "wallet" and the second doesn't. (I tend to prefer full sentences, but this is a mix in any casse.)